bevy_rapier3d/examples/ray_casting3.rs

```diff
@@ -88,7 +87,10 @@ pub fn cast_ray(
     // We will color in read the colliders hovered by the mouse.
     for (camera, camera_transform) in &cameras {
         // First, compute a ray from the mouse position.
-        let Some(ray) = camera.viewport_to_world(camera_transform, cursor_position) else {
+        let Some(ray) = camera
+            .viewport_to_world(camera_transform, cursor_position)
+            .ok()
+        else {
```

Comment:
```suggestion
        let Ok(ray) = camera.viewport_to_world(camera_transform, cursor_position) else {
```
Would this work as well? Mainly suggesting it due to formatting 😆